app/containers/ItunesContainer/index.js

```diff
@@ -0,0 +1 @@
+export default function ItunesContainer() {}
```

Comment:
integrate just the searching functionality. So the saga/reducer will be integrated. The part of rendering the cards in a grid will be taken in the next PR

Reply:
Understood, Searchbox feature to be implemented

Reply:
Updated in the PR accordingly